!SESSION 2020-03-13 14:03:46.882 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/users/kevpy/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/users/kevpy/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2020-03-13 14:04:51.699
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more

!ENTRY org.eclipse.cdt.core 1 0 2020-03-13 14:10:00.487
!MESSAGE Indexed 'usb_kb_bsp' (80 sources, 62 headers) in 2.07 sec: 1,190 declarations; 2,486 references; 110 unresolved inclusions; 6 syntax errors; 191 unresolved names (4.9%)

!ENTRY org.eclipse.cdt.core 1 0 2020-03-13 14:10:04.754
!MESSAGE Indexed 'usb_kb' (0 sources, 0 headers) in 0 sec: 0 declarations; 0 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0%)

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 14:22:20.481
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 14:22:22.775
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 4 0 2020-03-13 14:55:08.595
!MESSAGE Unable to validate connection settings.
!STACK 0
java.lang.Exception: Connected system ID hash not found on target at expected base address.
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.assertTrue(SystemConsoleConnectionModel.java:645)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateSystemId(SystemConsoleConnectionModel.java:589)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateConnectionSettings(SystemConsoleConnectionModel.java:511)
	at com.altera.sbtgui.launch.hardware.AbstractNios2CLaunchDelegate.launch(AbstractNios2CLaunchDelegate.java:115)
	at com.altera.sbtgui.launch.hardware.Nios2HardwareLaunchDelegate.launch(Nios2HardwareLaunchDelegate.java:98)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.altera.sbtgui.launch 4 0 2020-03-13 14:55:08.629
!MESSAGE Unable to validate connection settings.
!STACK 0
java.lang.Exception: Connected system ID hash not found on target at expected base address.
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.assertTrue(SystemConsoleConnectionModel.java:645)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateSystemId(SystemConsoleConnectionModel.java:589)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateConnectionSettings(SystemConsoleConnectionModel.java:511)
	at com.altera.sbtgui.launch.hardware.AbstractNios2CLaunchDelegate.launch(AbstractNios2CLaunchDelegate.java:115)
	at com.altera.sbtgui.launch.hardware.Nios2HardwareLaunchDelegate.launch(Nios2HardwareLaunchDelegate.java:98)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 14:55:47.199
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 14:55:49.413
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 15:18:47.972
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-13 15:18:50.267
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:03:36.242
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:03:38.825
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:09:38.326
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:09:40.603
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:11:44.500
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:11:46.819
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:17:12.480
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:17:15.844
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:22:29.684
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:22:37.705
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:22:44.559
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 4 0 2020-03-14 13:22:48.352
!MESSAGE Unable to validate connection settings.
!STACK 0
java.lang.Exception: Connected system ID hash not found on target at expected base address.
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.assertTrue(SystemConsoleConnectionModel.java:645)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateSystemId(SystemConsoleConnectionModel.java:589)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateConnectionSettings(SystemConsoleConnectionModel.java:511)
	at com.altera.sbtgui.launch.hardware.AbstractNios2CLaunchDelegate.launch(AbstractNios2CLaunchDelegate.java:115)
	at com.altera.sbtgui.launch.hardware.Nios2HardwareLaunchDelegate.launch(Nios2HardwareLaunchDelegate.java:98)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.altera.sbtgui.launch 4 0 2020-03-14 13:22:48.386
!MESSAGE Unable to validate connection settings.
!STACK 0
java.lang.Exception: Connected system ID hash not found on target at expected base address.
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.assertTrue(SystemConsoleConnectionModel.java:645)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateSystemId(SystemConsoleConnectionModel.java:589)
	at com.altera.embeddedsw.flash.model.internal.SystemConsoleConnectionModel.validateConnectionSettings(SystemConsoleConnectionModel.java:511)
	at com.altera.sbtgui.launch.hardware.AbstractNios2CLaunchDelegate.launch(AbstractNios2CLaunchDelegate.java:115)
	at com.altera.sbtgui.launch.hardware.Nios2HardwareLaunchDelegate.launch(Nios2HardwareLaunchDelegate.java:98)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:23:18.621
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584124747]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:23:25.514
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584124747 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:34:31.142
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584210563]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:34:38.047
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584210563 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:43:44.940
!MESSAGE Terminated Launch with connection: 10USB-Blaster on localhost [USB-0]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:43:48.616
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584210563]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 13:43:55.484
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584210563 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:12:22.423
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584210563]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:12:24.716
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584210563 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:17:53.788
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584210563]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:17:56.028
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584210563 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:28:02.791
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --stop --sidp=0x58 --id=0x0 --timestamp=1584210563]

!ENTRY com.altera.sbtgui.launch 1 0 2020-03-14 15:28:05.048
!MESSAGE Executing: [C:/users/kevpy/quartus/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=USB-Blaster on localhost [USB-0]' --device=1 --instance=0 --go --sidp=0x58 --id=0x0 --timestamp=1584210563 /cygdrive/c/Users/KevPy/Downloads/lab8_provided_bat/software/usb_kb/usb_kb.elf]
!SESSION 2020-03-21 14:43:01.544 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/users/kevpy/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/users/kevpy/quartus/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2020-03-21 14:48:58.602
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:176)
	at com.google.gson.Gson.fromJson(Gson.java:803)
	at com.google.gson.Gson.fromJson(Gson.java:768)
	at com.google.gson.Gson.fromJson(Gson.java:717)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:374)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:165)
	... 9 more
